components/taggroup/metadata/taggroup.yml

```diff
@@ -1,31 +1,40 @@
 name: Tag group
 status: Verified
-description: A group of tags.
+description: A group of <a href="https://opensource.adobe.com/spectrum-css/tag.html">tags</a>. Tags allow users to categorize content. They can represent keywords or people, and are grouped to describe an item or a search request. When horizontal space is limited in a tag group, the individual tags wrap to form another line.
 SpectrumSiteSlug: https://spectrum.adobe.com/page/tag/
 sections:
+  - name: Custom Properties API
+    description: |
+      This component can be modified via its `--mod-*` prefixed custom properties. A list of those prefixed custom properties can be found <a href="https://github.com/adobe/spectrum-css/tree/main/components/taggroup/metadata/mods.md">here</a>.
   - name: Migration Guide
     description: |
+      ### Updated tag classes
+      The examples now use the latest markup from the tag component. This includes additional classes:
+      - `spectrum-Tag-itemLabel`
+      - `spectrum-Tag-itemIcon`
+      - `spectrum-Tag-clearButton`
+
       ### A div wrapper is required for avatar
       ```
       <div class="spectrum-Avatar spectrum-Avatar--size50 spectrum-Tag-avatar">
```

Comment:
you didn't add this but, can we remove this class `spectrum-Tag-avatar` I am not seeing it in the `index.css` for `Tag`